components/mpas-ocean/bld/build-namelist

```diff
@@ -591,6 +586,9 @@ add_default($nl, 'config_GM_EG_riMin');
 add_default($nl, 'config_GM_EG_kappa_factor');
 add_default($nl, 'config_GM_EG_Rossby_factor');
 add_default($nl, 'config_GM_EG_Rhines_factor');
+add_default($nl, 'config_GM_horizontal_taper');
+add_default($nl, 'config_GM_horizontal_ramp_min');
+add_default($nl, 'config_GM_horizontal_ramp_max');
```

Comment:
this file and the next 3, I believe, should also contain the new namelist options for `GM_Rossby_ramp`

Reply:
That didn't come in with the automated tool...